### ISSUE TITLE
Distributed tracing support

### ISSUE THREAD

**Issue by TomKaltz** [dashersw/cote#77]
Zipkin integration would be very helpful for testing and monitoring.

**Comment by dashersw** [comment]
Yes, that would be amazing. I can help if anyone wants to tackle it.

**Comment by robodude666** [comment]
Perhaps this could be implemented via middleware #22?

There are many other tracing tools out there; I don't know if `cote` should support zipkin natively vs being an external plugin.

**Comment by dashersw** [comment]
Yes, this should ideally be a middleware, with a little bit more magic to make sure you don’t have to include it in every single requester / responder / etc. handler.
